fallible/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
Can you please add a repository key to the toml (or, if it's a direct copy from upstream, to avoid editing it) a README linking to the source repo?

Reply:
It is from https://github.com/servo/servo/tree/master/components/fallible. I'll add a README.